src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,31 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.common.Messages;
+import seedu.addressbook.data.person.ReadOnlyPerson;
+import seedu.addressbook.data.person.UniquePersonList.PersonNotFoundException;
+
+
+/**
+ * Deletes a person identified using it's last displayed index from the address book.
+ */
```

Comment:
Header comment not updated, this is still fine since it's a comment but surprisingly many problems in software come from careless copypasta